packages/components/src/internal/components/picklist/actions.ts

```diff
@@ -479,8 +487,7 @@ export const getPicklistFromId = async (listId: number): Promise<Picklist> => {
 
     const listSampleTypeData = await selectRows({
         schemaName: SCHEMAS.PICKLIST_TABLES.SCHEMA,
-        sql: 'SELECT DISTINCT SampleID.SampleSet as SampleType FROM "' + picklist.name + '"',
-        saveInSession: true,
+        sql: `SELECT DISTINCT SampleID.SampleSet as SampleType FROM "${picklist.name}"`,
```

Comment:
Hmm.  I wonder why the `saveInSession` was there. Probably a copy-paste remnant.